### PR TITLE
Epmcuxda 6740  top level menu

### PR DESCRIPTION
**Pull request by Leuvarden**
Added top level menu from screen: https://epam.invisionapp.com/d/main#/console/17142229/357988796/preview

- moved registration flow to separate RegistrationNavigator.js;
- created AppNavigator with SwitchNavigator, so we can't move back to registration/login form after successfully logging in;
- moved Observation screen to the separate stackNavigator: MainPageNavigator.js
- created Settings page (no real functionality, layout&styles only), moved it to the separate SettingsNavigator,
- components Birds and Logout in MainPageNavigator.js is for testing and to be replaces with actual screens

### CODE REVIEW

File: src/screens/Observations/Observations.js [hunk 1]
```diff
@@ -24,8 +26,12 @@ const Observations = () => {
 };
 
 Observations.navigationOptions = {
-  title: "Наблюдения",
-  headerRight: <DeleteObservation />
+  title: "Наблюдения"
```
Comment (AfanasyShevtsov) [inline src/screens/Observations/Observations.js]: Should we localize it?

File: src/screens/TopLevelMenu/TopLevelMenu.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+const logo = require("./../../assets/logotype/logotype2x.png");
+
+const TopLevelMenu = props => (
+  <SafeAreaView style={styles.safeArea}>
+    <View style={styles.imageContainer}>
+      <Image resizeMode="contain" style={styles.image} source={logo} />
+    </View>
+    <View>
+      <Text style={styles.userNameText}>Иван Иванов</Text>
```
Comment (AfanasyShevtsov) [inline src/screens/TopLevelMenu/TopLevelMenu.js:16]: Can we display user name here now?

File: src/screens/Settings/Settings.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+        <Text style={styles.hintText}>{translate("settings.email")}</Text>
+        <View style={styles.inputContainer}>
+          <Input
+            placeholder=""
+            label={translate("settings.email")}
+            labelStyle={styles.inputLabel}
+          />
+        </View>
+        <View style={styles.buttonContainer}>
```
Comment (AfanasyShevtsov) [inline src/screens/Settings/Settings.js]: We have 3 refresh buttons. Could we move it into a separate function?

File: src/screens/TopLevelMenu/TopLevelMenu.js (hunk 2 above)
Comment (Leuvarden) [reply src/screens/TopLevelMenu/TopLevelMenu.js:16]: I think it's better to fix it in scope of task with back-end integration

File: src/screens/Settings/Settings.js (hunk 3 above)
Comment (Leuvarden) [reply src/screens/Settings/Settings.js]: fixed it!

**Pull request merged**